src/renderer/components/Chart/index.js

```diff
@@ -159,7 +159,7 @@ export default function Chart({
               zeroLineColor: theme.text.shade10,
             },
             ticks: {
-              beginAtZero: true,
+              beginAtZero: false,
```

Comment:
:smooth:

Reply:
it's probably false by default but this seemed nicer :trollface: